communication/mav_modes.hpp

```diff
@@ -121,6 +121,14 @@ typedef enum
     CUST_GPS_BAD = 2048                         ///< GPS loss flag
 } mav_mode_custom_t;
 
+typedef enum 
+{
+    DUBIN_INIT = 0,
+    DUBIN_CIRCLE1 = 1, 
+    DUBIN_STRAIGHT = 2,
+    DUBIN_CIRCLE2 = 3,
```

Comment:
A Dubin trajectory is composed of a first circle, a straight line and a final circle. These three entities are computed at the start of the Dubin trajectory. We need the three part because the transition between them is different. (see https://github.com/lis-epfl/MAVRIC_Library/blob/cpp_quad_to_wing/communication/mavlink_waypoint_handler.cpp#L2269)